packages/jmix-react-core/src/app/Screens.test.ts

```diff
@@ -69,7 +69,7 @@ describe('Screens', () => {
       expect(screens.screens).toEqual([screen1]);
     });
 
-    it('saves history', () => {
+    xit('saves history', () => {
```

Comment:
Why do we disable this test?

Reply:
Because this functionality was commented out for now